lms/templates/dashboard.html

```diff
@@ -272,9 +272,11 @@ <h3>${course_dir}</h3>
 
         % if settings.FEATURES.get('ENABLE_ANNOUNCEMENTS'):
```

Comment:
@kaizoku Wow, weird that you had to do this, but I guess it's something about the template `<%block>` tags?

To make this look less confusing, could you move this line down to avoid wrapping this in two `% if settings.FEATURES.get('ENABLE_ANNOUNCEMENTS')` blocks? i.e.

```
          <%block name="skip_links">
            % if settings.FEATURES.get('ENABLE_ANNOUNCEMENTS'):
              <a class="nav-skip sr-only sr-only-focusable" href="#announcements" id="announcements-skip">${_("Skip to list of announcements")}</a>
            % endif
          </%block>
        % if settings.FEATURES.get('ENABLE_ANNOUNCEMENTS'):
          <%include file='dashboard/_dashboard_announcements.html' />
        % endif
```

Reply:
Yeah, I guess the blocks are rendered prior to the conditionals being evaluated or something?
This is especially strange because it's contrary to the Mako docs:
> Blocks also work in iterations, conditionals, just like defs:
> ```
> % if some_condition:
>     <%block>condition is met</%block>
> % endif
> ```

Though I guess this is not true for named blocks. I just tested, and while regular blocks work as expected in conditionals, named blocks don't.

Your refactor is definitely clearer though, I've changed it to match that pattern. Thanks!